lib/field-mapper.js

```diff
@@ -89,7 +89,7 @@ var amendMappingsBasedOnNyplSource = (data, nyplSource) => {
  */
 function buildMapper (type, nyplSource) {
   if (['bib', 'item'].indexOf(type) >= 0) {
-    let data = requireRemote(`https://raw.githubusercontent.com/NYPL/nypl-core/master/mappings/recap-discovery/field-mapping-${type}.json`)
+    let data = requireRemote(`https://raw.githubusercontent.com/NYPL/nypl-core/master/mappings/recap-discovery/field-mapping-${type}.json#v1.1`)
```

Comment:
Great idea to link to a specific version! Is this the right syntax? I thought it was something like:

https://raw.githubusercontent.com/NYPL/nypl-core/v1.0/mappings/recap-discovery/field-mapping-bib.json

Is there a v1.1 tag? I'm assuming you're planning on making one?

Also, maybe the version should be specified in an environment variable (or configuration file)? It might help with testing new versions.

Reply:
Yeah, I have the URL structure wrong. Thanks for catching it. My bad.

I had not pushed the update to `nypl-core` — wanted to get this a bit further down the pipe before doing that. It's out now. As you say, the correct URL is https://raw.githubusercontent.com/NYPL/nypl-core/v1.1/mappings/recap-discovery/field-mapping-bib.json

Storing the version in a variable makes sense — I prefer a configuration file for something like this. I'll take a look.

Thanks!